Expensify/App
# Create a local proxy server + npm script

Adds a local proxy server to get around cors issues + some webpack configs

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/146052

### Tests
1. First verify you can run `npm run web` without any issues
1. Copy the `.env.production` variables into `.env`
1. Set `EXPENSIFY_URL_COM` to be empty like so
```
EXPENSIFY_URL_COM=
EXPENSIFY_URL_CASH=https://www.chat.expensify.com/
```
4. `npm i`
5. `npm run proxy`
6. Verify you can sign in with your production user name
7. Verify Pusher is working
8. Verify you can leave a comment
9. Verify you can view images in chats
10. Verify you can upload images and files to chats

### Screenshots
<Add any necessary screenshots for all platforms if your change added or updated UI>
